frontend/src/css/modules/leaderboard.scss

```diff
@@ -1,9 +1,22 @@
 /* Styles for leaderboard page */
+a:hover {
```

Comment:
<img width="837" alt="Screenshot 2019-06-23 at 1 18 35 PM" src="https://user-images.githubusercontent.com/11664259/59980300-73744980-95b9-11e9-8053-7c25bfbd5608.png">

This impacts at other places too. assign a unique id to the tag and use that to assign styles.

Reply:
oops!!! my bad, sure will do that

Reply:
@galipremsagar done